web/packages/teleport/src/Integrations/IntegrationList.tsx

```diff
@@ -388,6 +389,30 @@ const IconCell = ({ item }: { item: IntegrationLike }) => {
   );
 };
 
+const DetailsCell = ({ item }: { item: IntegrationLike }) => {
+  let formattedText: string;
+
+  switch (item.resourceType) {
+    case 'integration':
+      switch (item.kind) {
+        case 'aws-oidc':
+          formattedText =
+            'Enroll EC2, RDS and EKS resources or enable Web/CLI access to your AWS Account.';
+          break;
+      }
+  }
+
+  if (!formattedText) {
+    return;
+  }
```

Comment:
we'd want to set `formattedText = item.details` by default, b/c `IntegrationList` actually displays both plugin type (which comes with the details field from backend) and integration type (which doesn't have this field set)

with that said though, do you mind i take over this? it looks like we now set details field for integration types [here](https://github.com/gravitational/teleport/blob/b4382d0dcb767c83109bd0ee0fec7254eff32cad/web/packages/teleport/src/services/integrations/integrations.ts#L434), but there is a bug there now so i have to fix it

Reply:
Sounds good 👍 
I'll assign the PR to you in that case